openwisp_radius/saml/views.py

```diff
@@ -66,6 +68,37 @@ def post_login_hook(self, request, user, session_info):
         try:
             user.registered_user
         except ObjectDoesNotExist:
+            email = None
```

Comment:
Instead of retrieving the email address from the SAML request, can we use the `user.email` field to get the email address? The field should get automatically populated by djangosaml2 using attribute mapping. Then, we can create the EmailAddress object with that value.